source/release-notes.txt

```diff
@@ -4,9 +4,22 @@ Release Notes
 
 .. default-domain:: mongodb
 
-MongoDB Connector for Spark 10.0.0
+MongoDB Connector for Spark 10.1.0
 ----------------------------------
 
-*Released on March 31, 2022*
+- Support for Scala 2.13.
+- Support for micro-batch mode with Spark Structured Streaming.
+- Support for BSON data types.
+- Improved partitioner support for empty collections.
+- Option to disable automatic upsert on write operations.
+- Improved schema inference for empty arrays.
+- Support for null values in arrays and lists. The Connector now stores these values
+  to MongoDB instead of throwing an exception.
```

Comment:
[suggestion] I feel like the preposition doesn't match the verb here. Maybe have it be "...writes these values to MongoDB..." or "...stores these values in MongoDB...".